### PR TITLE
Set Dev Genesis Starting Schema

### LINKED ISSUE #2126: Force Testnet on Paseo Schemas to Match Mainnet
With #2057, mainnet schemas are now included in the Genesis for the development builds of Frequency.

Should we apply mainnet schemas to testnet as well? YES

Would:
- Overwrite any schema ids that match mainnet schemas
- Need to be re-done with any new mainnet schema approval

Also:
- Start schema id at 16_000 for dev images and update testnet to also restart schema ids at 16_000

Would *not*:
- Overwrite schema names if additional schemas with the same name were added to testnet
    - Perhaps this is undesirable
- Overwrite any schemas greater than the max id of mainnet
- Be automated

How to doc: https://github.com/frequency-chain/frequency/wiki/Update-Testnet-and-Dev-Schemas-from-Mainnet

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is to update the development genesis to start at `16_000` so that schemas from mainnet do not collide.

Closes #2126 

# Discussion
- Update Genesis schemas from mainnet
- Updates to the state copy tool
- Set the Genesis value of `CurrentSchemaIdentifierMaximum` to `16_000`

# Checklist
- [x] Unit Tests added?
- [x] Spec version incremented?

# Expected:
- New schemas post merge should deploy to `16_000`+
- See these (in addition to other) mainnet schemas on dev. (State queries should match mainnet)
```
    dsnp.update: 19
    dsnp.reply: 18
    dsnp.tombstone: 16
    dsnp.broadcast: 17
```

### CODE REVIEW

File: resources/genesis-schemas.json
Comment (wilwade) [inline resources/genesis-schemas.json:1]: Updated via `make genesis-schemas`

File: tools/state-copy/remove.mjs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    if (page.length === 0) {
+      break;
+    }
+
+    startKey = page[page.length - 1].toString();
+  }
+  return result;
+}
+
+export async function remove(destUrl, storageKey, sudoUri = "//Alice", filterKeys = []) {
```
Comment (shannonwells) [inline tools/state-copy/remove.mjs:22]: nit: removeStorageKeys would be better IMO

File: tools/state-copy/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Tools to help copy state from one Frequency Chain to another.
+
+## Testnet Schemas Match
+
+To maintain alignment with Mainnet, when a new schema is deployed on Mainnet, Testnet data and Dev Genesis are set to have those exact same schemas.
+
+1. In the Frequency codebase: `cd tools/state-copy`
+2. `npm i`
+3. Edit `copy.mjs`
```
Comment (shannonwells) [inline tools/state-copy/README.md:11]:
```suggestion
3. Edit `schemas.mjs`
```

File: tools/state-copy/schemas.mjs [hunk 3]
```diff
@@ -1,18 +1,33 @@
...
+const DEST_URL = "ws://localhost:9944";
+
+// Mainnet
 const SOURCE_URL = "wss://1.rpc.frequency.xyz";
-const DEST_URL = "ws://127.0.0.1:9944";
 const STORAGE_KEY = "0xeec6f3c13d26ae2507c99b6751e19e76";
 const FILTER_OUT = [
   "0xeec6f3c13d26ae2507c99b6751e19e76d5d9c370c6c8aee1116ee09d6811b0d5", // governanceSchemaModelMaxBytes
   "0xeec6f3c13d26ae2507c99b6751e19e764e7b9012096b41c4eb3aaf947f6ea429", // palletVersion
+  // Comment this out if you INCLUDE setting the currentSchemaIdentifierMaximum from the SOURCE
```
Comment (shannonwells) [inline tools/state-copy/schemas.mjs:23]: how does one include this?

**Review by shannonwells — COMMENTED** [review]
I ran make start without the script to verify the genesis, I ran the script to fetch the state from testnet to localhost and that also worked.

I just have 2 nitpicks, the rename and the correction to the doc

File: tools/state-copy/remove.mjs (hunk 1 above)
Comment (wilwade) [reply tools/state-copy/remove.mjs:22]:
```suggestion
export async function removeStorageKeys(destUrl, storageKey, sudoUri = "//Alice", filterKeys = []) {
```

File: tools/state-copy/schemas.mjs (hunk 3 above)
Comment (wilwade) [reply tools/state-copy/schemas.mjs:23]: By commenting it out. I'm open to better wording as this is a . This is a filter list, so including it here, excludes it from the copy. Commenting in out removes it from the filter list, thus copying it.

Comment (JoeCap08055) [reply tools/state-copy/schemas.mjs:23]: Yeah, that is a bit confusing. How about:
```suggestion
  // Comment this out to INCLUDE setting the currentSchemaIdentifierMaximum from the SOURCE
```

**Review by shannonwells — APPROVED** [review]
Looks good to go, I don't think it's worth a respin just for the comment/instructions change unless you want to

File: tools/state-copy/schemas.mjs (hunk 3 above)
Comment (wilwade) [reply tools/state-copy/schemas.mjs]: Had to merge conflict the spec_version anyway, so set!

**Review by saraswatpuneet — APPROVED** [review]
read through files, looks good

**Review by mattheworris — APPROVED** [review]
- Read through code
- Ran tests (All passing)
🚢 it!

**Pull request merged**